sdks/go/test/integration/primitives/windowinto_test.go

```diff
@@ -36,3 +36,24 @@ func TestWindowSums_GBK(t *testing.T) {
 	WindowSums_GBK(s)
 	ptest.RunAndValidate(t, p)
 }
+
+func TestTriggerDefault(t *testing.T) {
+	integration.CheckFilters(t)
```

Comment:
This has been mentioned elsewhere, since we're using TestStream here, we need to filter these tests out from everything but the Flink runner. Fortunately, we can use regex and just cover the prefix "TestTrigger"